spec.md

```diff
@@ -692,3 +696,52 @@ Destructuring binding and assignment are not part of ES6, but all binding positi
 ```js
 interface Pattern <: Node { }
 ```
+
+# Concrete syntax
+
+Whitespace, comments, and the exact character sequences comprising Literal nodes are part of the lexical grammar, but not part of the _abstract_ syntax. Their representation is completely optional.
+
+```js
+interface ConcreteNode <: Node {
+    sourceElements: [ ChildReference | Token | Nontoken ];
```

Comment:
I did it this way because I don't think adding `sourceElements: [ ChildReference | Token | Nontoken ] | null` to `Node` would be valid syntax, but I see that I missed noting that everything else should extend not `Node` but `ConcreteNode`. I'll update.